### PR TITLE
Validate contact form

### PR DESCRIPTION
**Pull request by Rustamxon7**
We have created this pull request to check our validated contact form is working or not.
We have created a function to make a check if it is in lowercase or not. 
We have added an alert message if the email is not in lowercase.

Validate contact form

Feel free to review our PR and let us know if there are changes that need to be done!

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -134,3 +134,20 @@ popupOverlay.addEventListener('click', closePopUp);
...
+const msg = document.querySelector('.show-msg');
+
+function validationEmail(e) {
+  if (email.value !== email.value.toLowerCase()) {
+    msg.style.display = 'block';
+    e.preventDefault();
+  }
+}
+
+form.addEventListener('submit', validationEmail);
```
Comment (arslanbisharat) [inline script.js:151]: - [ ]  While I can see from the Gif you have added in the Pull Request message (Great Job!) you have implemented the validation rule but I cannot see any button on the screen (Live version). Please have a look at the screenshot below. 
FireShot Capture 002 - Personal Portfolio - raw githack com

Please troubleshoot this issue.

File: styles.css [hunk 2]
```diff
@@ -473,6 +473,48 @@ body {
...
+    visibility: hidden;
+    background-color: #fff;
+  }
+
+  to {
+    background-color: rgba(244, 244, 248, 0.048);
+  }
+}
+
+/* Alert message end */
```
Comment (arslanbisharat) [inline styles.css:516]: - [ ]  The alert message looking good but it should be near the `Submit `button and have a red color. This is how we find alerts on professionals sites. :smile:

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ]   Please add` .vscode` in `gitignore` file. Please have another look at ["How to make a professional-looking repository."](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css (hunk 2 above)
Comment (Rustamxon7) [reply styles.css:501]: This error has also been resolved. As you said, I changed the color of the alert messenger to red.

File: script.js (hunk 1 above)
Comment (Rustamxon7) [reply script.js:151]: I corrected the error you mentioned, you can check again. Thank you so much for pointing out the error. I was looking for this problem myself. Then I saw that it was a simple mistake !😁!

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**